pkg/apis/core/validation/validation.go

```diff
@@ -2840,6 +2840,20 @@ func validateProbe(probe *core.Probe, fldPath *field.Path) field.ErrorList {
 	return allErrs
 }
 
+func validateInitContainerRestartPolicy(restartPolicy *core.ContainerRestartPolicy, fldPath *field.Path) field.ErrorList {
```

Comment:
Generally we would evaluate `restartPolicy == nil` in this method by convention (see `validateProbe`).  Is there a reason *not* to follow that here?